tools/cli/commands-packages.js

```diff
@@ -1649,6 +1649,22 @@ main.registerCommand({
   } else {
     upgradePackageNames = options.args;
   }
+  var upgradePackagesWithoutCordova = _.reject(upgradePackageNames, function (name) {
+    return name.split(':')[0] === 'cordova'; 
+  });
+  if (upgradePackagesWithoutCordova !== upgradePackageNames) {
+    // There are some cordova packages in the list to update.
+    // We should tell users how to update cordova packages.
+    Console.warn('Looks like you\'re trying to update cordova packages.');
+    Console.warn('To update cordova packages, run:');
+    Console.warn('  meteor add PLUGIN@VERSION');
```

Comment:
It still doesn't make it very clear that you should use the `cordova:` prefix.  For example, with the code below, I might assume that I should run:

    meteor add cordova-plugin-inappbrowser@1.0.0

Which would not work.

Perhaps this should say:

    Console.warn('  meteor add cordova:PLUGIN-NAME@VERSION');